src/data/javascript-quiz.ts

```diff
@@ -75,6 +75,16 @@ const javascriptQuiz = [
       "Closures are special types of functions where the inner function will have access to variables in the outer function's scope.",
     Link: "https://www.freecodecamp.org/news/a-basic-guide-to-closures-in-javascript-9fc8b7e3463e",
   },
+  {
+    Question: "Which of the following scoping types does JavaScript use?",
+    Answer: "Lexical",
+    Distractor1: "Sequential",
+    Distractor2: "Segmental",
+    Distractor3: "Literal",
+    Explanation:
+      "JavaScript, like most current programming languages, employs lexical scoping. This means that functions are performed with the variable scope in effect when they were defined, rather than the variable scope in effect when they are invoked.",
+    Link: "https://www.freecodecamp.org/news/scope-and-closures-in-javascript",
+  },
```

Comment:
This question was removed for the quiz.
Please remove it here and make sure to update your main branch so you will have the most recent quiz files.